### PR TITLE
Update PR template to be slightly different

### PR DESCRIPTION
**Pull request by lukefrake**
Another attempt at a PR template, feedback so we can gauge opinions.

### CODE REVIEW

**Comment by kevinhodges** [comment]
+1

**Comment by notABluesSinger** [comment]
Does this PR require a Selenium test? Is this worded right for us. 
and after a recent IR, what about adding 'Will this affect multiple brands?'

**Comment by philmeehan** [comment]
:+1:

**Comment by lukefrake** [comment]
Merging this now as it's been up for about 21 hours without any negatives

File: prTemplate.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
-- [ ] Has this work been tested cross browser & cross device?
-- [ ] Would you +1 this PR if you were reviewing it?
\ No newline at end of file
+#### Reviewer 1
+- [ ] I agree with the assumptions made in the quality checklist.
+- [ ] I’ve witnessed the work behaving as expected.
+- [ ] I’ve witnessed the tests running succesfully.
+- [ ] I’ve checked for appropriate test coverage.
+- [ ] I’ve checked for coding anti-patterns.
+- [ ] I've checked this work against the requirements of the Jira.
```
Comment (panicnot) [inline prTemplate.md]: is there a reason why the last question is different for the two people?

Comment (lukefrake) [inline prTemplate.md]: Because nobody pointed it out before I merged!

Can you do a PR for this please!

**Pull request merged**